Gems/PhysX/Code/Tests/Benchmarks/PhysXSceneQueryBenchmarks.cpp

```diff
@@ -145,11 +145,11 @@ namespace PhysX::Benchmarks
         {
             request.m_direction = m_boxes[next].GetNormalized();
 
-            auto start = std::chrono::system_clock::now(); //AZStd::chrono::system_clock wont messeare below 1000ns
+            auto start = AZStd::chrono::system_clock::now(); //AZStd::chrono::system_clock wont messeare below 1000ns
 
             AzPhysics::SceneQueryHits result = sceneInterface->QueryScene(m_testSceneHandle, &request);
 
-            auto timeElasped = std::chrono::nanoseconds(std::chrono::system_clock::now() - start);
+            auto timeElasped = (AZStd::chrono::system_clock::now() - start);
```

Comment:
Should this line do a `duration_cast`?  Looks very similar to the 2 code blocks below.

Reply:
Yes it should be.